tests/net/ethernet_mgmt/testcase.yaml

```diff
@@ -0,0 +1,4 @@
+tests:
+  test:
+    min_ram: 32
+    tags: net vlan
```

Comment:
Take the vlan tag away here as there is nothing vlan specific in this test. Perhaps add ethernet_mgmt tag instead.